server/src/core/search/search_executor/postprocess.rs

```diff
@@ -244,3 +238,25 @@ fn generate_subtext(hit: &meilisearch::MSHit) -> String {
         None => building,
     }
 }
+
+fn unicode_split_at(search: &str, width: usize) -> (&str, &str) {
```

Comment:
I found a way, which is imo a bit cleaner. What do you think about this approach?

Reply:
Yeah that's good